goose-recipes/issue-details.yaml

```
@@ -15,6 +14,18 @@ parameters:
   description: A Jira issue (the Jira instance is choosen by the mcp server configuration)
 
 prompt: |
-  1. Open the defined Jira issue {{ issue }}.
-  2. Identify the repository (package) referred by the issue and confirm it exist in https://gitlab.com/redhat/centos-stream/rpms/(package), print the found url.
-  3. Identify the branch (for the previous found repository) where to work on, it depends on the fixVersion Jira value, verify the branch exists and print its name.
+  You are an AI Agent tasked to analyze Jira issues.  The issues usually describe a bug or issue around an RPM package or software component that must be updated in Red Hat Enterprise Linux.  Some issues are very explicit in what needs to be updated and to which version.  Others are more vague.  You can find information in the issue title, its description, fields and also in comments.  Make sure to take a close look before reporting the data.  Follow the following steps:
+
+  1. Open the {{ issue }} Jira issue.
+
+  2. Identify the name of the package that must be updated. Let's refer to it as `<package_name>`.
+     * Confirm the `<package_name>` repository exists by running `git ls-remote https://gitlab.com/redhat/centos-stream/rpms/<package_name>`. A successful command (exit code 0) confirms its existence.
+     * If the `<package_name>` does not exist, take another look at the Jira issue.  You may have picked the wrong package or name.
+
+  # TODO: should we make sure the version of the package exists upstream?
+  3. Identify the `<package_version>` the `<package_name>` should be updated or rebased to.
+
+  4. Identify the target branch `<git_branch>` of the `<package_name>` on GitLab to update. The branch information is usually the fixVersion Jira field.  Verify the branch exists on GitLab.
```

```suggestion
  4. Identify the target branch `<git_branch>` of the `<package_name>` on GitLab to update. The branch information is usually related with the fixVersion Jira field.  Verify the branch exists on GitLab.
```

I would not be too strict here, since the fixVersion is not exactly the branch name.